lib/core/Blockchain.ts

```diff
@@ -150,4 +164,23 @@ export default class Blockchain implements IBlockchain {
     console.info(`Refreshed blockchain time: ${responseBodyString}`);
     return responseBody;
   }
+
+  /**
+   * Gets the version information by making the REST API call.
+   */
+  private async tryGetServiceVersion() : Promise<ServiceVersionModel> {
+
+    try {
+      const response = await this.fetch(this.versionUri);
+
+      const responseBodyString = await ReadableStream.readAll(response.body);
+      console.info("Received version response from the blockchain service: ", responseBodyString);
+
+      return JSON.parse(responseBodyString);
+    } catch (e) {
+      console.error("Ignoring the exception during blockchain service version retrieval: %s", JSON.stringify(e));
```

Comment:
I'm surprised that linter didn't block double quotes, but we use [standardjs](https://standardjs.com/) style which uses single quotes.

In this case consider:

```suggestion
      console.error(`Ignoring the exception during blockchain service version retrieval: ${JSON.stringify(e)}`, );
```

Reply:
I would add that JSON.stringify doesn't work on error objects (returns `{}`) . 
You may want to consider using `JSON.stringify(err, Object.getOwnPropertyNames(err))` (see https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify)